Makefile

```diff
@@ -0,0 +1,158 @@
+VIRTUALENV=~/.venvs/pixua
+
+.PHONY: setup
+setup: requirements install
+
+.PHONY: requirements
+requirements: requirements_py requirements_js
+	@echo "Installed requirements"
+
+.PHONY: install
+install: requirements install_py install_js
```

Comment:
Not really, no. Since `setup` would only call `install`, it seems to me it will already have requirements in there. But you're right, if it'll only be run once, it doesn't matter much. But still, `setup` == `install`, so no need for requirements to be twice in `setup`.